### DISCUSSION TITLE
Why hasn't a new version been released after 10 months?

### DISCUSSION THREAD

**Discussion by liangrui1988** [apache/ozone#5435] [General]
Why hasn't a new version been released after 10 months? 1.3.0 No Related bug fixes?

Version	Release date
1.3.0	2022 Dec 18

**Comment by ChenSammi** [comment]
1.4.0 release is kicked off in the community. Currently, one of Ozone's major dependency, RATIS, is doing its 3.0.0 release.  So it might take 1~2 month to finish the whole Ozone 1.4.0 release.

**Reply by kerneltime** [reply]
@liangrui1988, in addition to what @ChenSammi mentioned. Many users branch out of the master and use it without waiting for a release. We aim to keep the master shippable.

**Reply by liangrui1988** [reply]
Ok, thank you.
